test/controllers/HomepageControllerSpec.scala

```diff
@@ -81,7 +83,7 @@ class HomepageControllerSpec extends FrontEndTestHelper {
     }
 
     "render the judgment homepage page with an authenticated judgment user" in {
```

Comment:
Can you add a test, if the `blockJudgmentPressSummaries` is `true`, then you shouldn't see those new contents?